src/js/views/awardsRecords.js

```diff
@@ -120,7 +105,7 @@ function getPlayerAwards(p, awardType) {
     const formatYear = year => {
         return Object.keys(year).map(k => {
             const years = helpers.yearRanges(year[k].map(y => y.season)).join(', ');
-            return `${k} <small>(${years})</small>`;
+            return `${k} (${years})`;
```

Comment:
Can you change it back to using `<small>`, just in case? Either make this a React component, or break up the `k` and `years` into separate variables and do it in AwardsRecords.js (the latter is probably preferable, unless it's complicated).